docs/extensions/reva/configuration.md

```diff
@@ -41,7 +41,7 @@ If you prefer to configure the service with commandline flags you can see the av
 
 Example service for Reva/oCIS
 
-Usage: `ocis-reva [global options] command [command options] [arguments...]`
+Usage: `reva [global options] command [command options] [arguments...]`
```

Comment:
it is a little bit annoying that we now have a `reva` binary, which does whet upstreams `revad` binary does ... and upstream has a  `reva` binary as well ,whic his the cli for the daemon ...

Reply:
I wonder if `storage` would make more sense ... similar to `web`